tests/reporters/unittest_reporting.py

```diff
@@ -208,37 +208,7 @@ def test_multi_format_output(tmp_path: Path) -> None:
         del linter.reporter
 
     with open(json, encoding="utf-8") as f:
-        assert (
-            f.read() == "[\n"
-            "    {\n"
-            '        "type": "convention",\n'
-            '        "module": "somemodule",\n'
-            '        "obj": "",\n'
-            '        "line": 1,\n'
-            '        "column": 0,\n'
-            '        "endLine": null,\n'
-            '        "endColumn": null,\n'
-            f'        "path": {escaped_source_file},\n'
-            '        "symbol": "missing-module-docstring",\n'
-            '        "message": "Missing module docstring",\n'
-            '        "message-id": "C0114"\n'
-            "    },\n"
-            "    {\n"
-            '        "type": "convention",\n'
-            '        "module": "somemodule",\n'
-            '        "obj": "",\n'
-            '        "line": 1,\n'
-            '        "column": 0,\n'
-            '        "endLine": null,\n'
-            '        "endColumn": null,\n'
-            f'        "path": {escaped_source_file},\n'
-            '        "symbol": "line-too-long",\n'
-            '        "message": "Line too long (1/2)",\n'
-            '        "message-id": "C0301"\n'
-            "    }\n"
-            "]\n"
-            "direct output\n"
-        )
+        assert '"messageId": "C0114"' in f.read()
```

Comment:
Why shorten the result?

Reply:
This doesn't need to test the full output, we only need to test that the output contains `"json"` output. That's the intent of the test.

If we want to test the format we should do so in the other tests that are specific to the json reporter (which this one is not).

Reply:
Thanks for explaining!